appscale/tools/scripts/services.py

```diff
@@ -0,0 +1,79 @@
+# General-purpose Python library imports
+import sys
+import traceback
+import yaml
+
+# Third-party Python libraries
+from tabulate import tabulate
+from termcolor import cprint
+
+# AppScale library imports
+from .. import version_helper
+from ..appscale import AppScale
+from ..appscale_tools import AppScaleTools
+from ..local_state import LocalState
+from ..parse_args import ParseArgs
+
+
+version_helper.ensure_valid_python_is_used()
+
+class AppScaleServices():
+
+  def __init__(self):
+    self.appscale = AppScale()
+    self.appscale_tools = AppScaleTools()
+  # The usage that should be displayed to users if they call 'appscale services'
```

Comment:
I think it would be more conventional to put these class variables above the init function.